src/CashuWallet.ts

```diff
@@ -1,39 +1,56 @@
+import {
+	blindMessage,
+	constructProofFromPromise,
+	serializeProof
+} from '@cashu/crypto/modules/client';
+import { deriveBlindingFactor, deriveSecret } from '@cashu/crypto/modules/client/NUT09';
+import { createP2PKsecret, getSignedProofs } from '@cashu/crypto/modules/client/NUT11';
+import { verifyDLEQProof_reblind } from '@cashu/crypto/modules/client/NUT12';
+import { hashToCurve, pointFromHex } from '@cashu/crypto/modules/common';
+import { DLEQ, type Proof as NUT11Proof } from '@cashu/crypto/modules/common/index';
```

Comment:
Why are there so many new imports in this file?

Reply:
these import already exist, my changes rearrange them in an alphabetical order, due to the prettier extension am using in my vscode